docs/howto_kubeconfig.md

```diff
@@ -0,0 +1,104 @@
+# Kubeconfig management
+Every time MicroShift starts it generates a set of kubeconfig files for local and remote management of the apiserver. These are tied to the certificates for the apiserver and are also impacted by MicroShift's configuration.
```

Comment:
It doesn't make new files every time, right? It updates them under some circumstances, but if they are present and valid it leaves them alone?

Reply:
File is created if it does not exist, but every time MicroShift starts it checks if it is up to date.
```bash
$ stat kubeconfig 
  File: kubeconfig
  Size: 5706      	Blocks: 16         IO Block: 4096   regular file
Device: fd00h/64768d	Inode: 33604994    Links: 1
Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
Context: system_u:object_r:container_var_lib_t:s0
Access: 2023-03-21 08:18:42.601264403 +0000
Modify: 2023-03-21 08:18:35.564777086 +0000
Change: 2023-03-21 08:18:35.564777086 +0000
 Birth: 2023-03-17 08:44:10.201752721 +0000
$ systemctl restart microshift
$ stat kubeconfig 
  File: kubeconfig
  Size: 5706      	Blocks: 16         IO Block: 4096   regular file
Device: fd00h/64768d	Inode: 33604994    Links: 1
Access: (0600/-rw-------)  Uid: (    0/    root)   Gid: (    0/    root)
Context: system_u:object_r:container_var_lib_t:s0
Access: 2023-03-21 08:25:28.343826507 +0000
Modify: 2023-03-21 08:25:21.306804581 +0000
Change: 2023-03-21 08:25:21.306804581 +0000
 Birth: 2023-03-17 08:44:10.201752721 +0000
```